spring-integration-core/src/test/java/org/springframework/integration/config/xml/StreamTransformerParserTests-context.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans:beans xmlns="http://www.springframework.org/schema/integration"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:beans="http://www.springframework.org/schema/beans"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+			http://www.springframework.org/schema/beans/spring-beans.xsd
+			http://www.springframework.org/schema/integration
+			http://www.springframework.org/schema/integration/spring-integration.xsd">
+
+	<channel id="directInput"/>
+
+	<channel id="queueInput">
+		<queue capacity="1"/>
+	</channel>
+
+	<channel id="output">
+		<queue capacity="1"/>
+	</channel>
+
+	<stream-transformer input-channel="directInput" output-channel="output"/>
+
+	<stream-transformer input-channel="queueInput" output-channel="output">
+		<poller fixed-delay="10000"/>
+	</stream-transformer>
+
+	<stream-transformer id="withCharset" charset="UTF-8"
+		input-channel="charsetChannel" output-channel="output"/>
```

Comment:
???
... **won't** be redundant 

I meant we should add the `<chain>` variant with the `<stream-transformer>` to this parser's test.

Note sure how to point easier for adding new changes to the file... 
That's why I did that in the end of this config.

Sorry, I need to be more clear.